### DISCUSSION TITLE
IT Playground - Mailchimp 적용하기

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-web#1047] [Blog]
# IT Playground - Mailchimp 적용하기

Quarto로 만드는 대시보드, 주요 기능, 소스코드 예시 등

https://it-playground.netlify.app/docs/blog/posts/2024-03-08-quarto-dashboard/

**Comment by worldinyj** [comment]
와우!!!!!

**Comment by cderv** [comment]
Please @worldinyj, do configure your website correctly - thank you !

See 
* https://github.com/holyrich/IT-Playground/issues/1
